dynparquet/reader.go

```diff
@@ -30,21 +30,23 @@ func ReaderFromBytes(buf []byte) (*SerializedBuffer, error) {
 }
 
 func NewSerializedBuffer(f *parquet.File) (*SerializedBuffer, error) {
+	b := &SerializedBuffer{
+		Reader: parquet.NewReader(f),
+		f:      f,
+	}
+
 	dynColString, found := f.Lookup(DynamicColumnsKey)
 	if !found {
-		return nil, ErrNoDynamicColumns
```

Comment:
I think I prefer empty dynamic columns than not having them at all. This has caught several bugs at this point, and I fear that gracefully accepting that there are no dynamic columns would hide bugs further.